### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by nils-werner-sonarsource**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
-.github/CODEOWNERS @sonarsource/sonarcloud
+.github/CODEOWNERS @Sonarsource/analysis-experience-squad
```
Comment (7PH) [inline .github/CODEOWNERS:2]: The org should be lowercase
```suggestion
.github/CODEOWNERS @sonarsource/analysis-experience-squad
```

**Review by 7PH — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (nils-werner-sonarsource) [reply .github/CODEOWNERS]: Looks like both do not work. Will get in touch with Dev-Infra/ITOPs

Comment (claire-villard-sonarsource) [reply .github/CODEOWNERS]: No worries - DevInfra needs to add some explicit permissions to solve the warning. I sent a request already for this repo https://sonarsource.atlassian.net/browse/BUILD-4707

**Review by claire-villard-sonarsource — APPROVED** [review]
LGTM

**Review by 7PH — APPROVED** [review]

**Pull request merged**